pom.xml

```diff
@@ -42,7 +42,7 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<java.version>1.6</java.version>
+		<java.version>1.8</java.version>
```

Comment:
That's the point: you can write everyting in Java 8, except for the core library, which is compiled with Java 6, and double checked to be compatible.